src/applications/caregivers/components/PreSubmitInfto/index.jsx

```diff
@@ -0,0 +1,188 @@
+    [signSignature, label, isSignatureComplete],
+  );
+
+  return (
+    <article className="signature-box">
+      {children && <header>{children}</header>}
+
+      <SignatureInput
+        setIsSigned={setIsSigned}
+        label={label}
+        fullName={fullName}
+      />
+
+      <ErrorableCheckbox
+        onValueChange={event => setIsChecked(event)}
+        label="Yes, the submitted information is true, accurate, and complete."
+      />
+    </article>
+  );
+};
+
+const PreSubmitCheckboxes = ({
+  onChange,
+  showError,
+  preSubmitInfo,
+  checked,
+  formData,
+}) => {
+  const [signatures, setSignature] = useState({
+    Veteran: false,
+    'Primary Caregiver': false,
+  });
+  const [secondaryCaregivers, setSecondaryCaregivers] = useState({
+    hasSecondaryOne: undefined,
+    hasSecondaryTwo: undefined,
+  });
+
+  useEffect(
+    () => {
+      const unSignedLength = Object.values(signatures).filter(
+        obj => Boolean(obj) === false,
+      ).length;
+
+      if (!unSignedLength) onChange(true);
+
+      const hasSecondaryOne =
+        formData?.secondaryOneFullName?.first &&
+        formData?.secondaryOneFullName?.last;
+
+      const hasSecondaryTwo =
+        formData?.secondaryTwoFullName?.first &&
+        formData?.secondaryTwoFullName?.last;
+
+      setSecondaryCaregivers({
+        hasSecondaryOne,
+        hasSecondaryTwo,
+      });
+    },
+    [checked, formData, onChange, preSubmitInfo, showError, signatures],
+  );
+
+  const CaregiverCopy = ({ label }) => {
+    const header = title => `${title} or Family Member Statement of Truth`;
+    return (
+      <>
+        <h3>{header(label)}</h3>
+
+        <p className="age-clause">
+          I certify that I am at least 18 years of age.
+        </p>
+
+        <p>
+          I certify that I am a family member of the Veteran or service member
+          named in this application OR I reside with the Veteran or service
+          member or will do so upon approval.
+        </p>
+
+        <p>
+          I agree to perform personal care services as the Primary Family
+          Caregiver for the Veteran or service member named on this application.
+        </p>
+
+        <p>
+          I understand that the Veteran may revoke my designation as Primary
+          Family Caregiver at any time and that the Secretary of the Department
+          of Veterans Affairs (or designee) may remove me from this position
+          immediately if I fail to comply with the Program requirements as
+          defined by law.
+        </p>
+
+        <p>
+          I understand that participation in the Program of Comprehensive
+          Assistance for Family Caregivers does not create an employment
+          relationship with the Department of Veterans Affairs.
+        </p>
+
+        <p>
+          I certify that the information above is correct and true to the best
+          of my knowledge and belief.
+        </p>
+      </>
+    );
+  };
+
+  /*
```

Comment:
Any of the code here and below will never be run since we [return early here](https://github.com/department-of-veterans-affairs/vets-website/pull/11352/files#diff-f094b34cb41735cddefd7376bf5aa1efR85), can we omit it? 🙂